Products/CMFPlone/relationhelper.py

```diff
@@ -85,55 +90,53 @@ def get_all_relations():
         try:
             rel = relation_catalog.resolveRelationToken(token)
         except ObjectMissingError:
-            logger.info(f'Token {token} has no object.')
+            logger.info(f"Token {token} has no object.")
             continue
 
-        if rel.from_object and rel.to_object:
-            try:
-                results.append({
-                    'from_uuid': rel.from_object.UID(),
-                    'to_uuid': rel.to_object.UID(),
-                    'from_attribute': rel.from_attribute,
-                })
-                info[rel.from_attribute] += 1
-            except AttributeError as ex:
-                logger.info(f'Something went wrong while storing {rel}: \n {ex}')
-        else:
-            logger.info(f'Dropping relation {rel.from_attribute} from {rel.from_object} to {rel.to_object}')
-    msg = ''
+        rel_uid_info = {
+            "from_attribute": rel.from_attribute,
+        }
+        rel_uid_info["from_uuid"] = rel.from_object and rel.from_object.UID() or None
+        try:
+            rel_uid_info["to_uuid"] = rel.to_object and rel.to_object.UID() or None
+        except AttributeError as e:
+            rel_uid_info["to_uuid"] = None
+            logger.error(f"Broken relation: {rel} '{str(e)}'")
```

Comment:
Would it be easier to see useful information in the log if this logged rel_uid_info instead of rel?

Reply:
Definitly.